modules/claims_api/app/sidekiq/claims_api/va_notify_follow_up_job.rb

```diff
@@ -19,10 +24,17 @@ class VANotifyFollowUpJob < ClaimsApi::ServiceBase
       slack_client.notify(msg)
     end
 
-    def perform(notification_id)
+    def perform(notification_id, poa_id)
       status = notification_response_status(notification_id)
       detail = "Status for notification #{notification_id} was '#{status}'"
 
+      # Call logic to map VANotify status to our internal step status
+      step_status = map_notify_status(status)
+      # Update the POA process step with latest status
+      poa = ClaimsApi::PowerOfAttorney.find(poa_id)
+      process = ClaimsApi::Process.find_or_create_by(processable: poa, step_type: 'CLAIMANT_NOTIFICATION')
+      process.update!(step_status:, error_messages: [], completed_at: Time.zone.now)
```

Comment:
I'm not sure we'd want to save the `completed_at` in every instance - only in success or failed cases, right? An "IN_PROGRESS" status can't be complete.

Reply:
Good catch, will fix.